presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanNodeStatsSummarizer.java

```diff
@@ -128,14 +130,16 @@ private PlanNodeStatsSummarizer() {}
             for (OperatorStats operatorStats : reverse(pipelineStats.getOperatorSummaries())) {
                 PlanNodeId planNodeId = operatorStats.getPlanNodeId();
 
-                // An "internal" pipeline like a hash build, links to another pipeline which is the actual output for this plan node
-                if (operatorStats.getPlanNodeId().equals(outputPlanNode) && !pipelineStats.isOutputPipeline()) {
-                    continue;
-                }
                 if (processedNodes.contains(planNodeId)) {
                     continue;
                 }
 
+                planNodeSpilledBytes.merge(planNodeId, operatorStats.getSpilledDataSize().toBytes(), Long::sum);
```

Comment:
but that moves the merge below the if-continue clause, I'd rather keep this here